frontend/src/fetchs/network.js

```diff
@@ -0,0 +1,14 @@
+
+const basePath = "http://localhost:4000/graphql"
+
+const headers = {
+  'Content-Type': 'application/json',
+  "x-api-key": "428bc20b5c5c7dec",
```

Comment:
We probably do not want to hard code this...

Reply:
Thinking this should come from the iframe data-ref

Reply:
Definitely. This is temporary code. We need to discuss the way to communicate between the iframe part and the main app. Remember that for now what we can the "iframe" is a separate page that we want to redirect from.